Dockerfile

```diff
@@ -28,9 +28,9 @@ RUN apt-get update && apt-get -y install \
 
 # Build and install CMake from source.
 WORKDIR /usr/src
-RUN git clone git://cmake.org/cmake.git CMake && \
+RUN git clone --branch v3.4.3 --depth 1 \
+     git://cmake.org/cmake.git CMake && \
```

Comment:
When the time will come, we could still clone the latest stable tag, and directly apply the patches of interest ?